exporter/datadogexporter/internal/logs/sender.go

```diff
@@ -71,8 +70,8 @@ func (s *Sender) SubmitLogs(ctx context.Context, payload []datadogV2.HTTPLogItem
 	// Correctly sets apiSubmitLogRequest ddtags field based on tags from translator Transform method
```

Comment:
We can move this `opts` variable from line 55 to be initialized here. This way, `s.opts.Ddtags` will always be reset on a new logs submission and will not be duplicated. If we do this, we can remove `opts` from lines 55 and 59 and it will work as intended.

```suggestion
	// enable sending gzip
	s.opts = *datadogV2.NewSubmitLogOptionalParameters().WithContentEncoding(datadogV2.CONTENTENCODING_GZIP)

	// Correctly sets apiSubmitLogRequest ddtags field based on tags from translator Transform method
```